pages/common/virt-clone.md

```diff
@@ -0,0 +1,8 @@
+# virt-clone
+
+> Clone a libvirt virtual machine.
+> More information: <https://virt-manager.org/>.
+
+- Clone a virtual machine generating a new virtual machine name and storage path from the original name and path, and a random MAC address:
+
+`virt-clone --original {{vm_name}} --auto-clone --mac RANDOM`
```

Comment:
The man page reads this about `--mac` (bold mine):

>**If this parameter is omitted**, **or** the value `RANDOM` is specified a suitable address will be randomly generated.

so what do you think about this?

```suggestion
`virt-clone --original {{vm_name}} --auto-clone`
```

Reply:
I think I've addressed this in the new commit.